package.json

```diff
@@ -1,41 +1,25 @@
 {
     "private": true,
+    "type": "module",
     "scripts": {
-        "dev": "npm run development",
-        "development": "mix && npm run ssr:build",
-        "fix:eslint": "eslint --ext .js,.vue resources/js/ --fix",
-        "fix:prettier": "prettier --write --loglevel warn 'resources/js/**/*.vue'",
-        "fix-code-style": "npm run fix:prettier && npm run fix:eslint",
-        "watch": "mix watch",
-        "watch-poll": "mix watch -- --watch-options-poll=1000",
-        "hot": "mix watch --hot",
-        "prod": "npm run production",
-        "production": "mix --production && npm run ssr:build",
-        "heroku-postbuild": "npm run prod",
-        "ssr:build": "mix --production --mix-config=webpack.ssr.mix.js",
-        "ssr:serve": "node public/js/ssr.js"
+        "dev": "vite",
+        "build": "vite build"
     },
     "dependencies": {
-        "@inertiajs/inertia": "^0.11.0",
-        "@inertiajs/inertia-vue3": "^0.6.0",
-        "@inertiajs/progress": "^0.2.7",
-        "@inertiajs/server": "^0.1.0",
+        "@inertiajs/progress": "^0.1.2",
```

Comment:
The latest version of inertia doens't need `@inertiajs/progress` to be installed:

https://inertiajs.com/upgrade-guide#progress